index.html

```diff
@@ -1,305 +1,295 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./styles.css" />
-    <title>Portfolio Setup And Mobile Version Skeleton</title>
-  </head>
-  <body>
-    <section class="wrapper">
-      <header class="header">
-        <div class="logo"><img src="./images/my-logo.svg" alt="My Logo" /></div>
-        <nav class="icon">
-          <img src="./images/icon-menu.svg" alt="Icon Menu" />
-        </nav>
-      </header>
-      <div class="main">
-        <h1><br>I’m George<br>Glad to see you!</h1>
-        <p>
-          I’m a software developer! I can help you build a product , feature or
-          website Look through <br>some of my work and experience! If you like what
-          you see and have a project you need<br> coded, don’t hestiate to contact
-          me.
-        </p>
-        <h3 class="connect">LET'S CONNECT</h3>
-        <ul>
-          <li>
-            <a href="#"><img src="./images/twitter.svg" alt="Twitter" /></a>
-          </li>
-          <li>
-            <a href="#"><img src="./images/linkedin.svg" alt="Linkedin" /></a>
-          </li>
-          <li>
-            <a href="#"><img src="./images/github.svg" alt="GitHub" /></a>
-          </li>
-          <li>
-            <a href="#"><img src="./images/angellist.svg" alt="Angellist" /></a>
-          </li>
-          <li>
-            <a href="#"><img src="./images/medium.svg" alt="Medium" /></a>
-          </li>
-        </ul>
-      </div>
-      <section class="works">
-        <div class="card">
-          <div class="card1">
-            <img src="./images/Snapshoot-Portfolio.svg" alt="Card 1" />
-          </div>
-          <h2>Tonic</h2>
-          <ul>
-            <li class="upper">Canopy</li>
-            <li>
-              <div class="counter"></div>
-            </li>
-            <li>Back End Dev</li>
-            <li>
-              <div class="counter"></div>
-            </li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <div class="nav-link">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-          </div>
-          <div class="see-project">
-            <a href="#">See Project</a>
-          </div>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./css/style.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+    <header class="navbar">
```

Comment:
- [ ] Don't you think like me? adjusting your padding-top for the header can make it more pretty, the space shown is too large, please reduce it a bit.
- [ ] 
<img width="502" alt="headersec" src="https://user-images.githubusercontent.com/79058364/155614814-8cced04f-b522-4d0d-bab0-66272c4b1da0.png">

Reply:
@BenMukebo 

Thanks for your feedback. I'm working on it